libraries/bsg_manycore.cpp

```diff
@@ -1363,22 +1363,22 @@ static int hb_mc_manycore_recv_read_rsp(hb_mc_manycore_t *mc,
         }
 
         /* check that the npa matches? */
-
+        
+        int shift = CHAR_BIT * (hb_mc_npa_get_epa(npa) & 0x3);
```

Comment:
Technically CHAR_BIT is the number of bits in a byte on the host machine ;-)

Reply:
So, sizeof(char)*8?

Reply:
Nah, it's fine as is! I'm just giving a hard time.